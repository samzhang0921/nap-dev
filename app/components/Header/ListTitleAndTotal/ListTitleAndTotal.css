.ListTitleAndTotal {
  order: 2;
  text-align: center;
  align-self: baseline;
}

h1 {
  text-align: center;
  width: 250px;
  padding-bottom: 4px;
  padding-top: 2px;
  line-height: 21px;
  text-transform: uppercase;
  font-size: 21px;
  font-family: ModernNAP-Display, serif;
  letter-spacing: 1px;
  background-color: transparent;
  color: #000;
  min-height: 20px;
  word-wrap: break-word;
  margin: 25px auto 0;
}

.resultsBox {

    text-align: center;
  margin-bottom: -8px;
  font-family: GillSans,"Gill Sans MT",Calibri,sans-serif;
  font-size: 14px;
}

.resultsBox span {
  padding: 0 15px;
  color: #666;
  background-color: white;
}
@media (max-width: 949px) {
  .ListTitleAndTotal {
    order: 1;
    width: 100%;
    margin-bottom: 10px;
  }
}
