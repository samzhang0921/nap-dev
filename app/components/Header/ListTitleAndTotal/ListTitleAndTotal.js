import React from 'react';
import style from './ListTitleAndTotal.css';
const ListTitleAndTotal = props => {
  return (
    <div className={style.ListTitleAndTotal}>
      <h1>what's new</h1>
      <div className={style.resultBox}>
        <div className={style.resultsBox}>
          <span>
            {props.total}&nbsp;Results
          </span></div>
      </div>
    </div>
  );
};

export default ListTitleAndTotal;
