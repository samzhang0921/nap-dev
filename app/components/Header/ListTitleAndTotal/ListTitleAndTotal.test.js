import React from 'react';
import {mount} from 'enzyme';
import toJson from 'enzyme-to-json';
import renderer from 'react-test-renderer';
import ListTitleAndTotal from './ListTitleAndTotal';

describe('ListTitleAndTotal', () => {

  const total =9;


  it('should mount item ', () => {
    const tree = toJson(mount(<ListTitleAndTotal />));
    expect(tree).toMatchSnapshot();
  });

  it('should match its snapshot with items', () => {

    const tree = renderer.create(
    <ListTitleAndTotal total={total}/>
  ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
