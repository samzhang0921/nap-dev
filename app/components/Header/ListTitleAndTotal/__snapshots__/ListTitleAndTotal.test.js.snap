// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListTitleAndTotal should match its snapshot with items 1`] = `
<div
  className="ListTitleAndTotal"
>
  <h1>
    what's new
  </h1>
  <div
    className="resultBox"
  >
    <div
      className="resultsBox"
    >
      <span>
        9
         Results
      </span>
    </div>
  </div>
</div>
`;

exports[`ListTitleAndTotal should mount item  1`] = `
<ListTitleAndTotal>
  <div
    className="ListTitleAndTotal"
  >
    <h1>
      what's new
    </h1>
    <div
      className="resultBox"
    >
      <div
        className="resultsBox"
      >
        <span>
           Results
        </span>
      </div>
    </div>
  </div>
</ListTitleAndTotal>
`;
